client/src/app/components/mission/mission.component.css

```diff
@@ -2,3 +2,26 @@
     display: flex;
     align-items: center;
 }
+
+a {
+  color: coral;
+  font-weight: 600;
+}
+
+.big_picture {
+  height: 90vh;
+  width: 100vw;
```

Comment:
+  height: 90vh;
+  width: 100vw;

this is not responsive (try making browser window smaller), it repeats, needs to be fixed

this might be a place where setting height in `vh` is warranted, but there should be no reason to set width at all, is by default 100%